wdn/templates_4.1/less/layouts/footer.less

```diff
@@ -55,7 +55,7 @@
         }
     }
 
-	span[class*='wdn-icon-'][aria-hidden='true']:before {
+	span[class*='wdn-icon-'][aria-hidden="true"]:before {
```

Comment:
Good call. I've applied them to `class`, `role`, and `type` attribute selectors, not just ARIA states.